### PR TITLE
Improve the landing page during vessel searches in vessel viewer // VV-451

### PR DESCRIPTION
**Pull request by fertandil87**
https://globalfishingwatch.atlassian.net/browse/VV-451

image

### CODE REVIEW

**Review by rdgfuentes — CHANGES_REQUESTED** [review]
LGTM however I could not find where are we displaying a different message when the vessel search timed out 
> If the search times out, it should say “the search has timed out, please try again

File: apps/vessel-history/features/search/search.selectors.ts [hunk 1]
```diff
@@ -4,6 +4,8 @@ import { selectAdvancedSearchFields, selectQueryParam } from 'routes/routes.sele
 import { getSerializedQuery } from './search.thunk'
 
 export const selectVesselsFound = (state: RootState) => state.search.queries
+export const selectSearchError = (state: RootState) => state.search.error
+export const selectSearchSources = (state: RootState) => ['AIS']
```
Comment (rdgfuentes) [inline apps/vessel-history/features/search/search.selectors.ts:8]: Why is this selector returning always `['AIS']`? shouldn't it get the sources from the API response?

Comment (fertandil87) [reply apps/vessel-history/features/search/search.selectors.ts:8]: ups, I added to test, nice catch kkk

**Review by rdgfuentes — APPROVED** [review]
👍

**Pull request merged**